odm_sdk/settings/user.py

```diff
@@ -26,6 +26,29 @@ def _get_server_url(host):
         "".format(host))
 
 
+def _read_non_canonical(msg):
```

Comment:
please add a small description wtf is that :)

Reply:
😬